### PR TITLE
Mobile version skeleton

### PR DESCRIPTION
**Pull request by Nedramrez**
- Set up a new repository and prepare it for development
- Add the toolbar section
- Add the headline section
- Add images
- Style the content

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,4 +1,339 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  position: absolute;
+  width: 375px;
+  height: 604px;
+  left: calc(50% - 375px / 2);
+  top: 92px;
+  background: url("assets/bg.png") repeat-y;
```
Comment (Mre55) [inline styles.css]: For the background image please make sure you stick to the Figma design and it's should cover the whole width of the screen as in the design. Kindly, fix that here I attached your design along with the Figma design to show how the background image.

| Your Design | Figma Design |
|-------------|---------------|
| image | image |

File: styles.css [hunk 2]
```diff
@@ -1,4 +1,339 @@
...
+}
```
Comment (Mre55) [inline styles.css]: Kindly, fix the horizontal scroll which I attached below in the video. I highly recommend not to use fixed-width as you do here in your CSS ☝️. If you remove the fixed width the horizontal scroll will be resolved.

https://user-images.githubusercontent.com/34578631/170269542-58a688e7-c53b-49e0-b72c-11335a48fd22.mov

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @Nedramrez,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nedramrez** [comment]
@Mre55  I fixed the background image and deleted the width property. Please check the code.

**Review by Mre55 — APPROVED** [review]
**Hi @Nedramrez,**

# Status: APPROVED 🏆 🎉 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

Alt Text


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**